provisions/group_vars/all

```diff
@@ -19,6 +19,7 @@ db_name: cccp
 db_pass: cccp
 db_host: '0.0.0.0'
 db_port: 5432
+postgresql_image: registry.centos.org/sclo/postgresql-95-centos7
```

Comment:
+1 for parameterising this

Reply:
Isn't this container image moving to `centos` namespace?